image-streams/imagesignature.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: ImageSignature
+metadata:
+ name: sha256:29f5d56d12684887bdfa50dcd29fc31eea4aaf4ad3bec43daf19026a7ce69912@imagesignaturetest2
```

Comment:
I think we could use <imagename> to instead of sha256:29f5d56d12684887bdfa50dcd29fc31eea4aaf4ad3bec43daf19026a7ce69912 .  That makes sense.